### DISCUSSION TITLE
Unable to Upload Objects in Bucket

### DISCUSSION THREAD

**Discussion by Aashiqps** [apache/cloudstack#9488] [Q&A] [answered]
I have created Object storage using Minio and is able to create bucket, but unable to upload objects in the bucket, getting below error. using the same access and secret key i am able to create bucket in other providers (cloudberry , cyberduck) and objects are getting uploaded.

Upload Failed
The request signature we calculated does not match the signature you provided. Check your key and signing method.

**Comment by rajujith** [comment, marked as answer]
@Aashiqps  this article might help you: https://www.shapeblue.com/hosting-a-static-website-using-cloudstack-and-minio/
